boards/esp32s2-mini/doc.txt

```diff
@@ -0,0 +1,201 @@
+/*
```

Comment:
The documentation has to be modified completely for the board.

Reply:
Yes, that's why it's still WIP :sweat_smile: